.changeset/shiny-ravens-lay.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Given the importance of this PR, let's add a changelog so it's easier for us to track it :)